sigstore/_internal/oidc/oauth.py

```diff
@@ -43,13 +43,64 @@
+          </svg>
+        </a>
+      </div>
+      <div>
+        <div class="title">
+          <span class="sigstore">sigstore </span>
+          <span>authentication successful!</span>
+        </div>
+        <div class="content">
+          <span>You may now close this page.</span>
+        </div>
+      </div>
+      <div class="anchor">
+        <div class="links">
+          <a href="https://sigstore.dev/" class="link login"><span class="sigstore">sigstore</span> home <span class="arrow">→</span></a>
+          <a href="https://docs.sigstore.dev/" class="link login"><span class="sigstore">sigstore</span> documentation <span class="arrow">→</span></a>
+          <a href="https://blog.sigstore.dev/" class="link"><span class="sigstore">sigstore</span> blog <span class="arrow">→</span></a>
+        </div>
+      </div>
+    </div>
+    <script>
+      document.getElementById("favicon").setAttribute("href", "data:image/svg+xml," + encodeURIComponent(document.getElementById("logo").outerHTML));
+    </script>
+  </body>
 </html>
-"""
+"""  # noqa: E501
```

Comment:
I've just decided to silence Ruff here as I don't think it helps readability to insert line breaks.